### ISSUE TITLE
Envoie de mail

### ISSUE THREAD

**Issue by lefevre00** [lefevre00/parking#101]
Ca risque de ne plus marcher hors Heroku.

Voir les conditions d'utiilisation de SendGrid pour usage interne.
J'imagine qu'il y a la possibilité de créer un nouveau compte SendGrid pour le serveur hébergé.

**Comment by vwilliam** [comment]
Pour le serveur en standalone, il faut implémenter un envoi de mail en SMTP vers localhost basique.

**Comment by lefevre00** [comment]
Oui, il va falloir mettre un contexte Spring par mode de serveur.
Interface de MailService avec impléméntation selon context :
- Standalone => service basé smtp
- heroku => service SendGrid
- test => service avec dump dans la console

**Comment by lefevre00** [comment]
En partie réglé.
Pour le version standalone, il va falloir mettre une config en place.
- Sur le serveur de recette/prod, envoie de mail
- Sur le tomcat du dev + jetty => log console
